README.md

```diff
@@ -31,18 +31,36 @@ Each team should fill in the blanks <<>> and add additional content that the tea
 Our product will be delivered using an Agile methodology that embraces the DevOps culture. Please note that our culture embraces change and these documents are treated as living, breathing artefacts that will be continuously updated.
 
 # Scrum Master
-<<list week number, student name>>
+Week 1 - 
+Week 2 - 
+Week 3 - 
+Week 4 - 
+Week 5 - 
+Week 6 - 
+Week 7 - 
+Week 8 - 
+Week 9 - 
+Week 10 - 
+Week 11 - 
+Week 12 - 
+Week 13 - 
+Week 14 - 
+Week 15 - 
 Rotating scrum master so that everyone gets to experience the role once.
 
 # Product Owner
 <<List name>> (Permanent)
```

Comment:
Who is the product owner here?

Reply:
Ill assume it is the repo owner so is that me?